samples/matter/light_bulb/README.rst

```diff
@@ -50,6 +50,8 @@ It can be tested in the following ways:
 The remote control testing requires either commissioning by the Matter controller device into a network or using the test mode.
 Both methods can be enabled after :ref:`building and running the sample <matter_light_bulb_sample_remote_control>`.
 
+The sample can be configured to use the secure bootloader and utilize it for performing over-the-air Device Firmware Upgrade using Bluetooth LE.
```

Comment:
This sentence is not needed, as it is repeated at the beginning of the "Device Firmware Upgrade support" section.